data/herg_karim_et_al/meta.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: herg_karim_et_al
+description: "A integrated Ether-à-go-go-related gene (hERG) dataset consisting\nof molecular structures labelled as hERG (<10uM) and non-hERG (>=10uM)\
+    \ blockers in\nthe form of SMILES strings was obtained from the DeepHIT, the BindingDB database,\nChEMBL bioactivity database, and other literature."
```

Comment:
What was our decision on line breaks? I'm in principle ok with them, but I think you used to remove them.

Reply:
Ahh, this was introduced in commit 53c5d3347c46c310663645b0aa39adfd2fb22cfd  when I changed a to à.
When `à` is removed again it works as expected.

Reply:
There are two options I see:
1. keep `a` because it seems that this is also widely used in the internet sources (maybe very likely to processing issues there? ;-) ), or
2. we get rid of the `\n` later because we will need an additional cleaning step anyway?

Reply:
Note / reminder: This applies to all the other hERG datasets.

Reply:
shall we still fix this?

Reply:
So if we strip the line breaks we are fine.